### PR TITLE
Add initialize step in Field Data docs

### PR DESCRIPTION
**Pull request by hpohekar**
Added `solver.solution.initialization.hybrid_initialize()` step.

otherwise we get,

```
>>> field_data = solver.field_data
>>> field_data.is_data_valid()                                                                            
False
```

### CODE REVIEW

**Review by mkundu1 — APPROVED** [review]

File: doc/source/api/solver/fielddata.rst [hunk 1]
```diff
@@ -19,6 +19,7 @@ available (for example, by reading case and data files):
   >>> import_filename = examples.download_file("mixing_elbow.msh.h5", "pyfluent/mixing_elbow")
   >>> solver = pyfluent.launch_fluent(mode="solver")
   >>> solver.file.read(file_type="case", file_name=import_filename)
+  >>> solver.solution.initialization.hybrid_initialize()
```
Comment (seanpearsonuk) [inline doc/source/api/solver/fielddata.rst:22]: We should now also adjust the above comment for consistency with the example code: "(for example, either by reading a case file and then initializing as in the following code, or by reading case and data files)".

Comment (hpohekar) [reply doc/source/api/solver/fielddata.rst:22]: We are not reading data files separately here.

We are using `solver.solution.initialization.hybrid_initialize()` in following example.

https://fluent.docs.pyansys.com/version/dev/examples/00-fluent/mixing_elbow_settings_api.html#sphx-glr-examples-00-fluent-mixing-elbow-settings-api-py

Comment (hpohekar) [reply doc/source/api/solver/fielddata.rst:22]: @seanpearsonuk  Earlier I thought we should be consistent with this example - https://fluent.docs.pyansys.com/version/dev/examples/00-fluent/mixing_elbow_settings_api.html#sphx-glr-examples-00-fluent-mixing-elbow-settings-api-py

Therefore I added above line only.  

but we are mentioning using both case and data files here.

image

Therefore we will add both initialize and data file reading steps.

Comment (seanpearsonuk) [reply doc/source/api/solver/fielddata.rst:22]: The code and comment should correlate better.  The comment is saying for example do this, but them the code is doing something different. Experts don't need the comment but non-experts just need clear comments that correlate with the code.

Comment (hpohekar) [reply doc/source/api/solver/fielddata.rst:25]: Done. Thank you.

File: doc/source/api/solver/fielddata.rst [hunk 2]
```diff
@@ -21,6 +24,17 @@ available (for example, by reading case and data files):
...
+
+.. code-block:: python
+
+  >>> import ansys.fluent.core as pyfluent
+  >>> from ansys.fluent.core import examples
+  >>> import_filename = examples.download_file("mixing_elbow.msh.h5", "pyfluent/mixing_elbow")
+  >>> data_filename = examples.download_file("mixing_elbow.dat.h5", "pyfluent/mixing_elbow")
+  >>> solver = pyfluent.launch_fluent(mode="solver")
+  >>> solver.file.read(file_type="case", file_name=import_filename)
+  >>> solver.file.read(file_type="data", file_name=data_filename)
```
Comment (seanpearsonuk) [inline doc/source/api/solver/fielddata.rst:37]: @hpohekar Now that this code has changed, it has become inconsistent with the changed comments.

Comment (hpohekar) [reply doc/source/api/solver/fielddata.rst:37]: @seanpearsonuk This is not clear to me.

Comment (seanpearsonuk) [reply doc/source/api/solver/fielddata.rst:37]: @hpohekar the above comment now reads "...(for example, either by reading a case file and then initializing as in the following code, ...)" which contradicts the "following code", which reading data instead.

Comment (hpohekar) [reply doc/source/api/solver/fielddata.rst:37]: Do we just want "(for example, either by reading a case file and then initializing as in the following code, or by reading case and data files)"  with previous version ? No need to add reading data file code ?

Comment (hpohekar) [reply doc/source/api/solver/fielddata.rst]: Done.

**Review by seanpearsonuk — APPROVED** [review]

**Review by raph-luc — APPROVED** [review]

**Pull request merged**